src/components/App.spec.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { shallow } from 'enzyme';
+
+import App from './App';
+
+const setup = (setupProps = {}) => {
```

Comment:
Maybe this is a redundant prop. We should declare it whenever we need, no need for placeholder here 😄. It should be applied to other places.